.github/workflows/starter.yml

```diff
@@ -0,0 +1,15 @@
+name: Workflow starter
```

Comment:
TBH, I'm not sure about the name of the file and the workflow 😉

Reply:
main_workflow? spawn_workflows?